packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -106,6 +106,13 @@ const validTransitions = {
  * }} MutableState
  */
 
+/**
+ * @typedef {{
+ *   state: ImmutableState & MutableState,
+ *   facets: *,
```

Comment:
```suggestion
 *   facets: {
 *     self: import('@agoric/vat-data/src/types').FunctionsMinusContext<typeof selfBehavior>,
 *     helper: import('@agoric/vat-data/src/types').FunctionsMinusContext<typeof helperBehavior>,
 *   },
```

Reply:
I thought this would be a straightforward edit, but it turned out to yield a bunch of follow-on type errors that I have not the first clue how to resolve.  I'm going to put it back the way it was and ask you to toss it in with all the other TS backfilling you've already flagged here.  Possibly also this could be added to the work in #5018?

Reply:
Ok